### PR TITLE
update styleguide

### PR DESCRIPTION
**Pull request by 105ron**
Add extra colors to the style guide.

### CODE REVIEW

File: app/views/static_pages/style_guide.html.erb [hunk 1]
```diff
@@ -47,4 +47,49 @@
...
+      </div>
+    </div>
+
+    <div class="col">
+      <div class="color-box">
+        <div class="color-preview black">
+          <p class="white">#fff</p>
+        </div>
+        <div class="color-name">
+          <p>$black</p>
```
Comment (arun1595) [inline app/views/static_pages/style_guide.html.erb]: Shouldn't this be $white?

File: app/views/static_pages/style_guide.html.erb [hunk 2]
```diff
@@ -47,4 +47,49 @@
...
+        <div class="color-name">
+          <p>$coffee</p>
+        </div>
+      </div>
+    </div>
+
+    <div class="col">
+      <div class="color-box">
+        <div class="color-preview black">
+          <p class="white">#fff</p>
```
Comment (arun1595) [inline app/views/static_pages/style_guide.html.erb]: Shouldn't this be #000?

**Comment by arun1595** [comment]
Nit. Add the `.white` class to all the parapgraphs displaying the color name except '#eee'

**Comment by KevinMulhern** [comment]
LGTM 👍 
:shipit:

**Pull request merged**